google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/features/ObjectApi.java

```diff
@@ -123,7 +124,7 @@
    @Fallback(NullOnNotFoundOr404.class)
    @Nullable
    GoogleCloudStorageObject getObject(@PathParam("bucket") String bucketName, @PathParam("object") String objectName,
-            GetObjectOptions options);
+         HttpRequestOptions options);
```

Comment:
I think GetObjectOptions is still useful because it provides GCE specific query params. Because it inherits from GetOptions which implements from HttpRequestOptions it can still be used for the getObject call here.